softdev.bib

```diff
@@ -2129,3 +2129,23 @@ @article{dijkstra1978gc
     year = {1978},
     pages = {966--975},
 }
+
+@inproceedings{barany2014python,
+    author = {Barany, Gerg\"{o}},
+    title = {Python Interpreter Performance Deconstructed},
+    booktitle = {Dynamic Languages and Applications},
```

Comment:
`booktitle = {DYLA}` (and remove `series`).

Reply:
Ok done.